### ISSUE TITLE
No op-ad-template support

### ISSUE THREAD

**Issue by pedrofelipe** [Automattic/fb-instant-articles#422]
# Steps required to reproduce the problem

I've found no way to use `op-ad-template`, since it's wrapping the whole `<section class="op-ad-template">`in a `<figure class="op-ad">` and `<iframe>` tag when I check on `/feed/instant-articles`.

This result in each ad slot being populated with all 4 ads I've put in a `op-ad-template`.

![screen shot 2016-08-25 at 21 25 13](https://cloud.githubusercontent.com/assets/463661/17990258/74531c50-6b0a-11e6-8a91-04f8f6fdb863.png)

When are you guys planning to support `op-ad-template`? I've seen that it is is listed on your Instant Articles [documentation](https://developers.facebook.com/docs/instant-articles/ads).
# Expected Result

```
<header>
...
  <section class="op-ad-template">
    <figure class="op-ad">
      <iframe src="url1" width="300" height="250" style="border:none;padding:0;margin:0;"></iframe>
    </figure>
    <figure class="op-ad">
      <iframe src="url2" width="300" height="250" style="border:none;padding:0;margin:0;"></iframe>
    </figure>
    <figure class="op-ad">
      <iframe src="url3" width="300" height="250" style="border:none;padding:0;margin:0;"></iframe>
    </figure>
    <figure class="op-ad op-ad-default">
      <iframe src="url4" width="300" height="250" style="border:none;padding:0;margin:0;"></iframe>
    </figure>
  </section>
</header>
```
# Actual Result

```
<header>
...
  <figure class="op-ad">
    <iframe width="300" height="250">
      <section class="op-ad-template">
        <figure class="op-ad">
          <iframe src="url1" width="300" height="250" style="border:none;padding:0;margin:0;"></iframe>
        </figure>
        <figure class="op-ad">
          <iframe src="url2" width="300" height="250" style="border:none;padding:0;margin:0;"></iframe>
        </figure>
        <figure class="op-ad">
          <iframe src="url3" width="300" height="250" style="border:none;padding:0;margin:0;"></iframe>
        </figure>
        <figure class="op-ad op-ad-default">
          <iframe src="url4" width="300" height="250" style="border:none;padding:0;margin:0;"></iframe>
        </figure>
      </section>
    </iframe>
  </figure>
</header>
```

**Comment by rossthedevigner** [comment]
@PedroFelipe Did you ever get this working?

**Comment by PedroFelipe** [comment]
@rossthedevigner nope :/

**Comment by diegoquinteiro** [comment]
The plugin does not suppor op-ad-template currently, so I've added the enhancement tag to the issue.

**Comment by diegoquinteiro** [comment]
This one is addressed in #296
